tests/llm/utils/classifiers.py

```
@@ -1,52 +1,25 @@
-from typing import Dict, List, Optional, Union, Literal
-from autoevals import Factuality, LLMClassifier
+from typing import Dict, List, Optional
```

Ruff (0.8.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)